db/warehouse/migrate/20250921222539_add_admin_editable_only_to_form_definition.rb

```diff
@@ -0,0 +1,13 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class AddAdminEditableOnlyToFormDefinition < ActiveRecord::Migration[7.0]
+  def change
+    add_column :hmis_form_definitions, :admin_editable_only, :boolean, default: false
```

Comment:
probably doesn't matter but maybe add null:false
```suggestion
    add_column :hmis_form_definitions, :admin_editable_only, :boolean, default: false, null: false
```